src/actions/groups.js

```diff
@@ -0,0 +1,163 @@
+    return async (dispatch, getState) => {
+        dispatch({type: GroupTypes.LINK_GROUP_SYNCABLE_REQUEST, data: {groupID, syncableID, type: syncableType}});
+
+        let data;
+        try {
+            data = await Client4.linkGroupSyncable(groupID, syncableID, syncableType, patch);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(batchActions([
+                {type: GroupTypes.LINK_GROUP_SYNCABLE_FAILURE, error, data: {groupID, syncableID, type: syncableType}},
+                logError(error),
+            ]));
+            return {error};
+        }
+
+        var type;
+        switch (syncableType) {
+        case Groups.SYNCABLE_TYPE_TEAM:
+            type = GroupTypes.LINKED_GROUP_TEAM;
+            break;
+        case Groups.SYNCABLE_TYPE_CHANNEL:
+            type = GroupTypes.LINKED_GROUP_CHANNEL;
+            break;
+        default:
+            console.warn(`unhandled syncable type ${syncableType}`); // eslint-disable-line no-console
+        }
+
+        dispatch(batchActions([
+            {type: GroupTypes.LINK_GROUP_SYNCABLE_SUCCESS, data: null},
+            {type, data},
+        ]));
+
+        return {data: true};
+    };
+}
+
+export function unlinkGroupSyncable(groupID: string, syncableID: string, syncableType: SyncableType): ActionFunc {
+    return async (dispatch, getState) => {
+        dispatch({type: GroupTypes.UNLINK_GROUP_SYNCABLE_REQUEST, data: {groupID, syncableID, type: syncableType}});
+
+        try {
+            await Client4.unlinkGroupSyncable(groupID, syncableID, syncableType);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(batchActions([
+                {type: GroupTypes.UNLINK_GROUP_SYNCABLE_FAILURE, error, data: {groupID, syncableID, type: syncableType}},
+                logError(error),
+            ]));
+            return {error};
+        }
+
+        var type;
+        var data = {group_id: groupID, syncable_id: syncableID};
+        switch (syncableType) {
+        case Groups.SYNCABLE_TYPE_TEAM:
+            type = GroupTypes.UNLINKED_GROUP_TEAM;
+            data.syncable_id = syncableID;
+            break;
+        case Groups.SYNCABLE_TYPE_CHANNEL:
+            type = GroupTypes.UNLINKED_GROUP_CHANNEL;
+            data.syncable_id = syncableID;
+            break;
+        default:
+            console.warn(`unhandled syncable type ${syncableType}`); // eslint-disable-line no-console
+        }
+
+        dispatch(batchActions([
+            {type: GroupTypes.UNLINK_GROUP_SYNCABLE_SUCCESS, data: null},
+            {type, data},
+        ]));
+
+        return {data: true};
+    };
+}
+
+export function getGroupSyncables(groupID: string, syncableType: SyncableType): ActionFunc {
+    return async (dispatch, getState) => {
+        dispatch({type: GroupTypes.GET_GROUP_SYNCABLES_REQUEST, data: {groupID, syncableType}});
```

Comment:
In the action above, it's `type: syncableType` but here it's just `syncableType`. Is there a reason for the inconsistency?